specification/chaos/resource-manager/Microsoft.Chaos/stable/2023-11-01/types/experiments.json

```diff
@@ -656,6 +651,85 @@
       },
       "additionalProperties": false
     },
+    "experimentExecutionDetails": {
+      "description": "Model that represents the execution details of an Experiment.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "String of the resource type.",
+          "readOnly": true
+        },
+        "id": {
+          "type": "string",
+          "description": "String of the fully qualified resource ID.",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "String of the resource name.",
+          "readOnly": true
+        },
+        "properties": {
+          "description": "The properties of the experiment execution details.",
+          "$ref": "#/definitions/experimentExecutionDetailsProperties",
+          "x-ms-client-flatten": true,
+          "readOnly": true
+        }
+      }
+    },
+    "experimentExecutionDetailsProperties": {
```

Comment:
make this definition extend experimentExecutionProperties and remove the properties defined in experimentExecutionProperties already.